.lando.yml

```diff
@@ -106,41 +106,41 @@ services:
     services:
       image: drupalci/webdriver-chromedriver:production
       command: chromedriver --log-path=/tmp/chromedriver.log --verbose --whitelisted-ips=
-  # elasticsearch:
-  #   type: compose
-  #   services:
-  #     image: "docker.elastic.co/elasticsearch/elasticsearch:7.17.0"
-  #     command: "/bin/tini -- /usr/local/bin/docker-entrypoint.sh eswrapper"
-  #     user: elasticsearch
-  #     environment:
-  #       ES_JAVA_OPTS: "-Xms512m -Xmx512m"
-  #       discovery.type: "single-node"
-  #       bootstrap.memory_lock: "true"
-  #       # Allow CORS requests.
-  #       http.cors.enabled: "true"
-  #       http.cors.allow-origin: "*"
-  #     ulimits:
-  #       memlock:
-  #         soft: "-1"
-  #         hard: "-1"
-  #     ports:
-  #       - "9200:9200"
-  #     volumes:
-  #       - esdata:/usr/share/elasticsearch/data
-  #   # Install ES plugins.
-  #   build_as_root:
-  #     - elasticsearch-plugin install analysis-icu analysis-ukrainian
-  #   volumes:
-  #     esdata:
-  #       driver: local
-  # kibana:
-  #   type: compose
-  #   services:
-  #     image: "docker.elastic.co/kibana/kibana:7.17.0"
-  #     command: "/bin/tini -- /usr/local/bin/kibana-docker"
-  #     user: kibana
-  #     ports:
-  #       - "5601:5601"
+  elasticsearch:
+   type: compose
+   services:
+     image: "docker.elastic.co/elasticsearch/elasticsearch:7.17.0"
```

Comment:
Why elasticsearch 8?

Reply:
I had the same idea, I tried initially to use elasticsearch 8 but it didn't work locally (can't remember the exact issue), and apparently it has not been used in projects yet, so I didn't want to expand the scope too much.

(discussion with Janne about this https://wunder.slack.com/archives/C049GMMBEAC/p1671794114416579 )

Reply:
Maybe we could have it as a followup task?